app/models/machine_condition.rb

```diff
@@ -17,7 +17,8 @@ def update(options = {})
     if options[:comment]
       self.comment = options[:comment]
 
-      if location_machine_xref.machine_conditions.first == self || (location_machine_xref.machine_conditions.size > 1 && location_machine_xref.machine_conditions.first == self)
+      # retain legacy "above the fold" comment until we can verify it is safe to remove from code
+      if location_machine_xref.machine_conditions.first == self
         location_machine_xref.condition = comment
         location_machine_xref.save
```

Comment:
was talking to @RyanTG about this and apparently he removed the frontend display code for this on the web... if it's also not used on the app it could probably be deleted as a column and this code could be removed